src/lib/staged_ledger_diff/body.ml

```diff
@@ -10,7 +10,7 @@ end
 module Stable = struct
   module V1 = struct
     type t = { staged_ledger_diff : Diff.Stable.V2.t }
-    [@@deriving compare, sexp, fields]
+    [@@deriving equal, compare, sexp, fields]
```

Comment:
Why was equality added to staged ledger diff bodies? I'm weary of adding this because, in practice, we should probably only be comparing staged ledger diffs by hashes and not using structural equality to compare them.

Reply:
This is used in test which ensures that the staged ledger diff read from DB is the same that created.